test/Passes/DefaultPipeline/tpp-mapping.mlir

```diff
@@ -1,13 +1,90 @@
-// RUN: tpp-opt %s -tpp-mapping -split-input-file | FileCheck %s
+// RUN: tpp-opt %s -tpp-mapping -cleanup -split-input-file | FileCheck %s
```

Comment:
Is this cleanup just for the test or is it really necessary? If the latter, then it should go in the pipeline after `createGeneralizeTensorPackAndUnPackPass`.

Reply:
It is already present in the default pipeline as we run cleanup after every major stage i.e., mapping, lowering etc.
It is mainly for the test as it produces a bit nicer IR. But you're right, it should be in the mapping pass to limit number of possible combinations.